portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -87,8 +87,8 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	<div class="pagination-bar" data-qa-id="paginator" id="<%= namespace + id %>">
 		<c:if test="<%= deltaConfigurable %>">
 			<div class="dropdown pagination-items-per-page">
-				<a class="dropdown-toggle page-link" data-toggle="liferay-dropdown" href="javascript:;"><liferay-ui:message arguments="<%= delta %>" key="x-entries" />
-					<span class="sr-only"><liferay-ui:message key="per-page" /></span>
+				<a class="dropdown-toggle page-link" data-toggle="liferay-dropdown" href="javascript:;">
+					<liferay-ui:message arguments="<%= delta %>" key="x-entries" /><span class="sr-only">&nbsp;<liferay-ui:message key="per-page" /></span>
```

Comment:
Hey @jorgediaz-lr, Can you use [`StringPool.NBSP`](https://github.com/liferay/liferay-portal/blob/c095d83e59e09f7f1e439efaba10a01a7b2e8785/modules/test/poshi/poshi-core/src/main/java/com/liferay/poshi/core/util/StringPool.java#L134) instead of hardcoding `&nbsp;` ? 💔

Reply:
Hi @diegonvs 

I have replaced the &nbsp; with StringPool.NBSP, see https://github.com/liferay-frontend/liferay-portal/pull/2062/commits/2ce3b280685b972e7414f74bfdee6fce95bd0bdc